config/dev.secrets.exs.example

```diff
@@ -0,0 +1,14 @@
+use Mix.Config
+
+# Configure Arc File Upload
+
+## Uncomment below to use local storage
+config :arc, storage: Arc.Storage.Local
+
+## Uncomment below to use AWS S3
+# config :arc,
+#   virtual_host: true
+#
+# config :ex_aws,
+#   access_key_id: "ACCESS_KEY_ID",
```

Comment:
This is what I meant can we use this instead! Really not a fan of storing keys in config files if we do not have too.
https://hexdocs.pm/ex_aws/1.1.4/ExAws.html
config :ex_aws,
  access_key_id: [{:system, "AWS_ACCESS_KEY_ID"}, :instance_role],
  secret_access_key: [{:system, "AWS_SECRET_ACCESS_KEY"}, :instance_role],

Reply:
In fact that has been added in https://github.com/source-academy/cadet/blob/master/config/config.exs#L37.
I added this line here to overwrite the config in development when we are not running on EC2.
Hope that clarifies.

Reply:
I think my point here is even on the local machine its best to store the keys in env vars or even better still within the aws cli tool with aws configure! Hope that clarifies!

Reply:
Okay get it! Will remove this line then.